inception-pdf-editor/src/main/java/de/tudarmstadt/ukp/inception/pdfeditor/pdfanno/PdfAnnoPanel.java

```diff
@@ -101,6 +107,41 @@ public void onRequest()
             }
         });
 
+        add(annoProvider = new AbstractDefaultAjaxBehavior() {
+
+            private static final long serialVersionUID = 3L;
+
+            @Override
+            protected void respond(AjaxRequestTarget aTarget)
+            {
+                SourceDocument doc = aModel.getObject().getDocument();
+
+                File pdfFile = documentService.getSourceDocumentFile(doc);
+
+                try
+                {
+                    String pdftext = PDFExtractor.processFileToString(pdfFile, false);
+                    AnnoFile annoFile = pdfAnnotationEditor.renderAnnoFile(pdftext);
+                    String script = "setTimeout(function() { " +
+                        "var annoFile = `\n" +
+                        annoFile +
+                        "`;\n" +
+                        "pdfanno.contentWindow.annoPage.importAnnotation({" +
+                        "'primary': true," +
+                        "'colorMap': " + annoFile.getColorMap().toString() + "," +
+                        "'annotations':[annoFile]}, true);" +
+                        "}, 10);";
+
+                    aTarget .appendJavaScript(script);
+                }
+                catch (IOException e)
+                {
+                    log.error("Unable to get PDF text for " + pdfFile.getName()
```

Comment:
Also add a use-visible error message by calling `org.apache.wicket.Component.error(Serializable)` and `target.addChildren(getPage(), IFeedback.class);`.